src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -175,7 +167,7 @@ namespace pluginhost {
             // Visual C++ debug runtime: we append to the plugin name. Must only
             // load debug plugins iff we're a debug server
             const auto isDebugRuntimePlugin =
-                boost::iends_with(pluginBaseName, PLUGIN_HOST_DEBUG_SUFFIX);
+                boost::iends_with(pluginBaseName, OSVR_PLUGIN_DEBUG_SUFFIX);
```

Comment:
I don't know the answer to this, but if we have two suffixes (`.debug` and `.manualload`), which order must they appear in if they're both used? Is that order imposed properly by the build tools? Do we want to support allowing the suffixes to appear in any order?

Reply:
Based on the latest OSVR Core snapshot, I'm seeing that plugins that are debug and manualload are named as follows: `com_osvr_pluginName.debug.manualload.dll` 

From  osvrAddPlugin.cmake script I confirmed that 
1. Plugin `SUFFIX` is set to `manualload.dll` (if it is a manualload plugin) which gets appended to the end of the target name (according to the <https://cmake.org/cmake/help/v3.6/prop_tgt/SUFFIX.html>)
2. Plugin `DEBUG_POSTIFX` is set to `.debug` which is appended to the target file name (https://cmake.org/cmake/help/v3.6/prop_tgt/CONFIG_POSTFIX.html)
Hence the order would always be as follows `com_osvr_pluginName.debug.manualload.dll` for debug, manualload plugins.